src/plugins/paginate/paginate-en.hbs

```diff
@@ -0,0 +1,839 @@
+				<p>Blanditiis, obcaecati quas sed consequuntur deserunt culpa odit, sint tempore soluta temporibus expedita veniam! Impedit?</p>
+			</div>
+		</article>
+		<article class="col-sm-6 col-md-4 col-lg-3">
+			<div class="well">
+				<h3 class="mrgn-tp-0">Debitis sint vel voluptas illo.</h3>
+				<p>Autem nam rerum omnis praesentium dicta, maxime maiores rem, perferendis, possimus qui vero ad sapiente.</p>
+			</div>
+		</article>
+		<article class="col-sm-6 col-md-4 col-lg-3">
+			<div class="well">
+				<h3 class="mrgn-tp-0">Quam dolores modi mollitia nesciunt.</h3>
+				<p>Suscipit rerum sunt, vel quam labore doloribus illo adipisci nam ea similique impedit maiores id.</p>
+			</div>
+		</article>
+		<article class="col-sm-6 col-md-4 col-lg-3">
+			<div class="well">
+				<h3 class="mrgn-tp-0">Possimus dolore nulla velit doloribus!</h3>
+				<p>Officiis pariatur nobis sequi quod culpa enim natus est praesentium, dolor sint voluptas, dignissimos eum.</p>
+			</div>
+		</article>
+		<article class="col-sm-6 col-md-4 col-lg-3">
+			<div class="well">
+				<h3 class="mrgn-tp-0">Assumenda, praesentium id. Alias, sint.</h3>
+				<p>Mollitia tempora quae optio dicta obcaecati. Consectetur nisi iste sunt, eos perspiciatis maxime at dignissimos?</p>
+			</div>
+		</article>
+		<article class="col-sm-6 col-md-4 col-lg-3">
+			<div class="well">
+				<h3 class="mrgn-tp-0">Earum officia saepe voluptatibus dolor!</h3>
+				<p>Eum non natus ullam, adipisci expedita, amet iusto quam, officiis ut error necessitatibus harum voluptatem?</p>
+			</div>
+		</article>
+		<article class="col-sm-6 col-md-4 col-lg-3">
+			<div class="well">
+				<h3 class="mrgn-tp-0">Impedit natus aperiam nemo adipisci?</h3>
+				<p>Illum dolore ipsam corporis atque quisquam cupiditate quod aliquid quaerat, saepe architecto aspernatur ratione autem!</p>
+			</div>
+		</article>
+		<article class="col-sm-6 col-md-4 col-lg-3">
+			<div class="well">
+				<h3 class="mrgn-tp-0">Assumenda dignissimos beatae doloremque dolores.</h3>
+				<p>Aliquid temporibus sunt quis illum, voluptatem atque. Blanditiis voluptatum iure cupiditate recusandae hic sint aperiam.</p>
+			</div>
+		</article>
+	</div>
+</section>
+
+<section id="filteredEventsList">
+	<h2>Example 4 - Events list</h2>
+	<p>This example leverages the following options: <code>itemsPerPage</code>, <code>section</code>, and <code>selector</code></p>
+	<p>This example also leverages the following other two plugins: <b>Filter</b> and <b>Tagfilter</b>. In order for all three plugins to work together, they have to be added to the same HTML element.</p>
+	<section class="wb-paginate wb-tagfilter provisional wb-filter"
+	data-wb-filter='{
+		"uiTemplate": "#filteredEventsList",
+		"section": ".events-list",
+		"selector": "article"
+	}'
+	data-wb-paginate='{
+		"itemsPerPage": 5,
+		"section": ".events-list",
+		"selector": "article"
+	}'>
+		<h3 class="wb-fltr-info h3"><span data-nbitem></span> event(s)</h3>
```

Comment:
Adding the role status overwrite the default role heading level 3 of the HTML element `<h3>`.

Here the heading element seem to be used for styling purpose only and not for semantic.